### ISSUE TITLE
Request: for Kotlin, offer remove un-needed `?` and `!!` when you see `?)!!` 

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [Miha-x64/Mikes_IDEA_extensions#19]
Often when I convert from Java to Kotlin, I see something like this:

```
foo((value as Boolean?)!!)
```

Instead, it should be like this:

```
foo(value as Boolean)
```

If `foo` is defined only as `foo(Boolean)` , it could even be replaced in this case to just:

```
foo(value)
```

Can you please add this ? I don't know of similar cases, but maybe there are some...

**Comment by Miha-x64** [comment]
Yes! Any chain with `?`s inside looks suspicious when ends with `!!`.

**Comment by AndroidDeveloperLB** [comment]
You say you know of other, similar cases?
